mesop/components/uploader/uploader.py

```diff
@@ -92,5 +104,27 @@ def uploader(
       )
       if on_upload
       else "",
+      type_index=_get_type_index(type),
+      type=type,
+      color=color,
+      disable_ripple=disable_ripple,
+      disabled=disabled,
     ),
+    style=style,
   )
+
+
+def _get_type_index(
+  type: Literal["raised", "flat", "stroked", "icon"] | None,
```

Comment:
Drop icon here and  on L128. You removed it in L76, which I think is right because icon button needs to have a child component, and this seems overtly complex for uploader right now.

Reply:
Good catch. Updated. 

I think I didn't do it on purpose. I just copied what was on button. So button is also the same where the interface does not include "icon" but the underlying code still has the icon option.